styles/_blog.less

```diff
@@ -230,3 +253,173 @@ article {
     display: inline;
   }
 }
+
+[class*=sqs-col] .front-page-search {
+  padding: 0;
+}
+
+.sqs-search-ui-button-wrapper .search-input {
+  background: url(/assets/search-icon.svg) no-repeat 15px 50%;
+  font-family: Sentinel A, Sentinel B;
+  font-size: 24px;
+  font-weight: 900;
+  line-height: 1.4;
+  padding: 12px 12px 12px 65px;
+  color: @badgerBlack;
+  background-color: white;
+  border: none;
+  opacity: 1;
+  -webkit-appearance: none;
+}
+
+.sqs-search-container input {
+  font-family: Sentinel A, Sentinel B;
+  font-size: 40px;
+  font-weight: 900;
+  line-height: 1.4;
+  padding: 12px 12px 12px 65px;
+  color: @badgerBlack;
+  border: none;
+  opacity: 1;
+  -webkit-appearance: none;
+}
+
+.sqs-search-page-input {
+  padding: 0px;
+  background: none;
+  margin-bottom: 0px;
+  border: none;
+  background: url(/assets/search-icon.svg) no-repeat 15px 50%;
+  background-color: white;
+  -webkit-appearance: none;
+}
+
+.sqs-spin.large {
+  height: 12px;
+  width: 12px;
+}
+
+.sqs-search-ui-button-wrapper .search-input::placeholder {
+  color: red;
+}
+
+.sqs-search-container-item .sqs-main-image-container {
+  display: none;
+}
+
+.sqs-search-preview-ui .sqs-search-ui-result .sqs-search-ui-list .search-result.clear {
+  margin: 0px;
+  padding: 0px;
+}
+
+input::-webkit-input-placeholder {
```

Comment:
this one will affect input fields everywhere on the whole site. Is this the intention?